content-services/7.3/admin/auth-sync.md

```diff
@@ -1465,6 +1465,20 @@ Configure the `alfresco-global.properties` file using the below properties:
 | identity-service.resource | The **Client ID** for the client created within your realm that points to Content Services. The default value  is `alfresco`. |
 | identity-service.public-client | The adapter won't send credentials for the client to the Identity Service if this is set to `true`. The default value  is `true`. |
 
+#### Enable configuration for Keycloak
+
+For Keycloak compatibility, you must set `Exclude Issuer from Authentication Response` to `On`.
+
+1. Log in to the Keycloak Admin Console.
+2. Select the **alfresco** realm.
+3. Navigate to the **Clients** section.
+4. Select the client that is used by Share according to the `aims.resource` property.
+
+> **Note:** When using the Identity Service for authentication, see [Configure Alfresco Share Properties]({% link identity-service/latest/tutorial/sso/ldap.md %}#step-5-configure-alfresco-share-properties). When using Keycloak for authentication, see .

```

Comment:
There is no reason to mention Identity Service here as this workaround is only necessary for vanilla Keycloak. Also Identity Service is reaching EoL in September 2024.